var/spack/repos/builtin/packages/sparrow/package.py

```diff
@@ -3,83 +3,23 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-import os
-
 from spack.package import *
 
 
-class Sparrow(CMakePackage):
-    """Sparrow: fast semiempirical quantum chemical calculations.
-
-    When publishing results obtained with Sparrow, please cite
-    the corresponding release as archived on Zenodo
-    (DOI 10.5281/zenodo.3244105; please use the DOI of the respective
-    release).
-
-    In addition, we kindly request you to cite the following article
-    when using Sparrow:
+class Sparrow(Package):
```

Comment:
The old package should be identical but with `deprecated=True` added to the version. That way users have at least 1 release cycle to switch to the new name without anything breaking.